functions/spanner/index.js

```diff
@@ -50,9 +50,7 @@ exports.get = async (req, res) => {
     const rows = results[0].map(row => row.toJSON());
     rows.forEach(row => {
       res.write(
-        `SingerId: ${row.SingerId}, ` +
-          `AlbumId: ${row.AlbumId}, ` +
-          `AlbumTitle: ${row.AlbumTitle}\n`
+        `SingerId: ${row.SingerId}, AlbumId: ${row.AlbumId}, AlbumTitle: ${row.AlbumTitle}\n`
```

Comment:
I believe @grant [was 👎on this](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/1489#discussion_r326799551).